src/pages/_app.tsx

```diff
@@ -114,12 +119,14 @@ const App: React.FC<React.PropsWithChildren<AppProps>> = ({
             <CioProvider>
               <QueryClientProvider client={queryClient}>
                 <TrpcProvider>
-                  <MDXProvider components={mdxComponents}>
-                    {getLayout(Component, pageProps)}
-                  </MDXProvider>
-                  <div className="print:hidden">
-                    <ReactQueryDevtools />
-                  </div>
+                  <PostHogProvider client={posthog}>
```

Comment:
This enables us to use their `usePosthog` hook and handle web tracking in our React components. Not using any of those features at the moment though.

Reply:
The providers pyramid grows ever taller